cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/co/cask/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -141,10 +142,20 @@ public String toString() {
       "} " + super.toString();
   }
 
+  /**
+   * @return a builder used to create the config for a data pipeline
+   */
```

Comment:
If the schedule is optional, then it seems that the pattern should be:
```
... = builder().
   ...
   .addTimeSchedule()
   ...
   .build(); 
```
Other than backward-compatibility, is there a reason not to have it that way?

Reply:
true, I'll add a `setTimeSchedule` (multiple time schedules are not supported).

Reply:
👍